src/main/c/output.c

```diff
@@ -605,3 +605,73 @@ jobject output_convert_schema_checked(JNIEnv *env, const ddwaf_object *obj)
     JNI(DeleteLocalRef, ret);
     return NULL;
 }
+
+jobject convert_ddwaf_object_to_jobject(JNIEnv *env, const ddwaf_object *obj)
+{
+    switch (obj->type) {
+        // TODO: Implement missing types, when needed
+        case DDWAF_OBJ_INVALID:
+        case DDWAF_OBJ_SIGNED:
+        case DDWAF_OBJ_UNSIGNED:
+        case DDWAF_OBJ_FLOAT:
+        case DDWAF_OBJ_BOOL:
+            return NULL;
```

Comment:
Are these never used?

Reply:
Answered offline: these are not used in 1.17.0, they will be used in later versions.